.idea/misc.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@desimal911 together with the file for the first task, you launched the IDE files, which should not exist here. Please, remove them from the request and leave only the task files.